src/main/java/com/regnosys/rosetta/common/transform/PipelineModel.java

```diff
@@ -0,0 +1,92 @@
+package com.regnosys.rosetta.common.transform;
+
+public class PipelineModel {
+
+    private String id;
+    private String name;
+    private Transform transform;
+    private String upstreamPipelineId;
+    private Serialisation outputSerialisation;
+
+    public PipelineModel(String id, String name, Transform transform, String upstreamPipelineId, Serialisation outputSerialisation) {
+        this.id = id;
+        this.name = name;
+        this.transform = transform;
+        this.upstreamPipelineId = upstreamPipelineId;
+        this.outputSerialisation = outputSerialisation;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Transform getTransform() {
+        return transform;
+    }
+
+    public String getUpstreamPipelineId() {
+        return upstreamPipelineId;
+    }
+
+    public Serialisation getOutputSerialisation() {
+        return outputSerialisation;
+    }
+
```

Comment:
all classes (including inner classes) should have `equals`, `hashCode` and `toString` (use Intellij to auto-generate these